sleuth-camel-core/src/main/java/com.playtika.sleuth.camel/CamelMessageTextMap.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * The MIT License (MIT)
```

Comment:
fyi while this is indeed targeted at sleuth 1.5 which won't add new features, if you later want to do a brave one, it would need to be licensed again as apache to merge into brave or sleuth

Reply:
yes, I'm aware, anyway code will be different, only approach will be the same